express-post-json/server.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+
+const app = express();
+
+const nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res, next) => {
+  res.json(Object.values(grades));
+  next();
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = req.body;
+  grades[nextId].id = nextId;
+  res.status(201).send(grades);
```

Comment:
The spec says to respond with just the currently-added grade, not with all of them. Please fix.